mesonbuild/compilers/swift.py

```diff
@@ -40,11 +40,17 @@ def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoic
                          linker=linker)
         self.version = version
 
+    def get_pic_args(self) -> T.List[str]:
+        return []
+
+    def get_pie_args(self) -> T.List[str]:
+        return []
+
     def needs_static_linker(self) -> bool:
         return True
 
     def get_werror_args(self) -> T.List[str]:
-        return ['--fatal-warnings']
+        return ['-warnings-as-errors']
```

Comment:
Was this something that changed during the lifetime of the Swift compiler or a bad copy-paste job during the original implementation of this file?

Reply:
It appears to have been copypasted from `ValaCompiler` in d92504797f50ea3a4abcd8f8d0ee1d903abca8af.  Google doesn't turn up any evidence that swiftc ever accepted `--fatal-warnings`.